pallets/staking/src/lib.rs

```diff
@@ -1748,7 +1748,7 @@ pub mod pallet {
             let ledger = ledger.rebond(value);
             // last check: the new active amount of ledger must be more than ED.
             ensure!(
-                ledger.active >= T::Currency::minimum_balance(),
+                ledger.active >= <ExistentialDeposit<T>>::get(),
```

Comment:
https://github.com/paritytech/substrate/blob/master/frame/staking/src/pallet/mod.rs#L1478
reference substrate `pallet-staking` it's no need to change `rebond`